packages/plugin-core/src/_extension.ts

```diff
@@ -425,6 +429,21 @@ export async function _activate(
     toggleViews(false);
   }
 
+  if (extensionInstallStatus === InstallStatus.INITIAL_INSTALL) {
+    const VimInstalled = VSCodeUtils.isExtensionInstalled("vscodevim.vim");
+    if (VimInstalled) {
+      AnalyticsUtils.track(ExtensionEvents.VimExtensionInstalledInitial);
+      const [keybindingConfigPath, resolvedKeybindings] = getVimExtensionResolvedKeybinding();
+      if (!_.isUndefined(resolvedKeybindings)) {
+        if (!fs.existsSync(keybindingConfigPath)) {
+          fs.ensureFileSync(keybindingConfigPath);
+          fs.writeFileSync(keybindingConfigPath, "[]");
+        }
+        writeJSONWithComments(keybindingConfigPath, resolvedKeybindings);
```

Comment:
can we also add another event here? eg.  track(VimExtensionInstalled, fixApplied: true)

Reply:
Will this be a separate count from the one just above? I don't see how much insight we can get from this unless we try to resolve keybinding conflicts on every `InstallStatus.UPGRADED` instead of `InstallStatus.INITIAL_INSTALL`.

Reply:
it could be that the user installed the extension but something went wrong with the fix. 

now that I think about it, this might change this to an error metric instead (jonathan is currently working on error reporting) in a future change